### DISCUSSION TITLE
SEDIT manual or instructions?

### DISCUSSION THREAD

**Discussion by Dietrich-L** [lynchaj/nhyodyne#51] [Q&A]
Does a manual or some instructions for SEDIT exist?

**Comment by lynchaj** [comment]
Hi
I think the sedit.asm source is available which might help. Maybe Dan Werner can help?

Thanks, Andrew Lynch
     On Wednesday, November 9, 2022, 03:24:16 PM EST, Dietrich-L ***@***.***> wrote:  
 
 


Does a manual or some instructions for SEDIT exist?

**Comment by Dietrich-L** [comment]
Well, the source is there and I ported it to my system.  Nevertheless a manual or command summary would be nice.
